### PR TITLE
Include the Dev Service image in the project

### PR DESCRIPTION
**Pull request by iocanel**

### CODE REVIEW

File: .github/workflows/docker.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+permissions:
+  attestations: write
+  id-token: write
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
```
Comment (gastaldi) [inline .github/workflows/docker.yaml:25]:
```suggestion
      uses: actions/checkout@v4
```

File: .github/workflows/docker.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
+name: Publish Docker
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Tag for the Docker image (optional)'
+        required: false
+  create:
```
Comment (gastaldi) [inline .github/workflows/docker.yaml:8]: I think it's `push`? 
```suggestion
  push:
```

Comment (iocanel) [reply .github/workflows/docker.yaml:8]: `create` worked fine in the original project.

File: .github/workflows/docker.yaml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+permissions:
+  attestations: write
+  id-token: write
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```
Comment (gastaldi) [inline .github/workflows/docker.yaml:22]: To create multiarch images shouldn't it run on different OSes too? Never done that before

Comment (iocanel) [reply .github/workflows/docker.yaml:22]: No, its not necessary.

Comment (iocanel) [reply .github/workflows/docker.yaml:22]: ... or at least this is how I got it working locally and in the original project.

**Review by gastaldi — APPROVED** [review]
LGTM, looking forward to seeing it in action

**Pull request merged**